source/Calamari.CloudAccounts/AzureOidcAccount.cs

```diff
@@ -41,21 +36,62 @@ public AzureOidcAccount(IVariables variables)
             TenantId = variables.Get(AccountVariables.TenantId);
             Jwt = variables.Get(AccountVariables.Jwt);
             AzureEnvironment = variables.Get(AccountVariables.Environment);
-            ResourceManagementEndpointBaseUri = variables.Get(AccountVariables.ResourceManagementEndPoint, DefaultVariables.ResourceManagementEndpoint);
-            ActiveDirectoryEndpointBaseUri = variables.Get(AccountVariables.ActiveDirectoryEndPoint, DefaultVariables.OidcAuthContextUri);
+            ResourceManagementEndpointBaseUri = variables.Get(AccountVariables.ResourceManagementEndPoint, DefaultAccountEndpoints.ResourceManagementEndpoint);
+            ActiveDirectoryEndpointBaseUri = variables.Get(AccountVariables.ActiveDirectoryEndPoint, DefaultAccountEndpoints.OidcAuthContextUri);
         }
 
         public AccountType AccountType => AccountType.AzureOidc;
         public string GetCredentials => Jwt;
-        public string SubscriptionNumber { get;  }
+
+        public string SubscriptionNumber { get; }
         public string ClientId { get; }
         public string TenantId { get; }
         public string Jwt { get; }
         public string AzureEnvironment { get; }
         public string ResourceManagementEndpointBaseUri { get; }
         public string ActiveDirectoryEndpointBaseUri { get; }
 
-        internal static string GetDefaultScope(string environmentName)
+        public async Task<string> GetAuthorizationToken(ILog log, CancellationToken cancellationToken)
```

Comment:
All of this code was in the two extension method classes (one for each type), but changed to instance methods so they can be accessed through the `IAzureAccount`